apps/vaporgui/MappingFrame.cpp

```diff
@@ -316,9 +327,9 @@ void MappingFrame::populateSamplingHistogram() {
     coords[Y] = minExts[Y];
     coords[Z] = minExts[Z];
 
-    int iSamples = SAMPLE_RATE;
-    int jSamples = SAMPLE_RATE;
-    int kSamples = SAMPLE_RATE;
```

Comment:
This code doesn't look correct. It appears to be doubling up on the stride, and not advancing the coordinates correctly.

More importantly, what is the purpose of populateSamplingHistogram()?

Reply:
But why is populateSamplingHistogram  needed? How is it different from populateIteratingHistogram()? I.e why do we need both?

Reply:
@clyne 

It's debatable whether we need it at this point.  Previously, we would compute our histograms based on the actual data volume within a renderer's given region.  This required sampling the grid for the SliceRenderer.  Using an iterator was not representative of the actual data volume of the slice.

Now that we're fudging the histogram with _stride, maybe it's reasonable to fudge it further by just gathering an iterator from a hyperslab that's adjacent to the current slice.  However this will only work if slices are axis aligned.  Leigh indicated that rotating slices would be an important thing to have for the slicer.